sparkles/tests/test_checks.py

```diff
@@ -40,7 +40,37 @@ def test_check_P2():
     assert 'less than 3.0 for ER' in msg['text']
 
 
-def test_guide_count_er():
+def test_n_guide_check_not_enough_stars():
+    """Test the check that number of guide stars selected is typical"""
```

Comment:
I think this is checking that the number of guide stars is as-requested.

Reply:
Fixed.